src/_colors.scss

```diff
@@ -35,6 +48,7 @@ $active-gray: hsla(0, 0, 0, .1);
 $active-dark-gray: hsla(0, 0, 0, .2);
 $box-shadow-gray: hsla(0, 0, 0, .25);
 $overlay-gray: hsla(0, 0, 0, .75);
+$transparent: rgba(229, 240, 254, 0);
```

Comment:
It's a barely perceptible blue, for the fade-out gradient at the bottom of the Invision mockup. I guess a better name for this might indicate its blueness?

![screen shot 2018-07-31 at 11 09 38 am](https://user-images.githubusercontent.com/3431616/43468542-45f18d14-94b2-11e8-8d68-a7833c6105f3.png)

Reply:
Oh interesting, I think I did the same thing by mixing some other colors together in [#1997](https://github.com/LLK/scratch-www/pull/1997/files#diff-72cf6e9b2977e6c0858134e338fb2026R174). We should maybe try and decide on which way is the right one?

Reply:
@benjiwheeler - Maybe we could grab 30min to pair on cleaning up these colors. 

Looks like there are a few places we have redundancies and other places we could consolidate down into using a core set of the color palette. Theoretically we shouldn't need to add this many new colors to the palette.